app/models/mixins/container_archived_mixin.rb

```diff
@@ -0,0 +1,14 @@
+module ContainerArchivedMixin
+  def archived?
+    ems_id.nil?
+  end
+
+  # Needed for metrics
+  def my_zone
+    if ext_management_system.present?
+      ext_management_system.my_zone
+    elsif old_ext_management_system.present?
```

Comment:
@kbrock  
Right now in the model there is `delegate   :my_zone, :to => :ext_management_system`
but this wont work for archived entities since they are disconnected from the `ems`.
this PR removes that delegate and puts this method in, instead.